llvm/test/Feature/Repo/repo_hash_graph_contributions.ll

```diff
@@ -0,0 +1,36 @@
+; Test GO's information of Contributions: load and store instructions.
+; A global variable `B` has a reference to another global variable `A`, `A` will be put into the Contributions of `B`.
```

Comment:
I think that this sentence could be simplified: ”global variable B has a reference to global variable A which will be a Contribution for B”. (Same applies to the other occurrences of similar phrases.)

Reply:
Changed in commit cca1f2d.